calendar-app/App.js

```diff
@@ -8,7 +8,6 @@ const App = () => {
   return (
     <SafeAreaProvider>
       <View style={styles.container}>
-        <Header />
```

Comment:
Create a bug in current sprint to fix this, since you said you will rework it when it is merged